README.md

```diff
@@ -1,2 +1,114 @@
-# class-data-inheritable
-Class::Data::Inheritable - Inheritable, overridable class data
+NAME
+
+`Class::Data::Inheritable` - Inheritable, overridable class data
+
+SYNOPSIS
+
+    package Stuff;
+    use base qw(Class::Data::Inheritable);
+
+    # Set up DataFile as inheritable class data.
+    Stuff->mk_classdata('DataFile');
+
+    # Declare the location of the data file for this class.
+    Stuff->DataFile('/etc/stuff/data');
+
+    # Or, all in one shot:
+    Stuff->mk_classdata(DataFile => '/etc/stuff/data');
+
```

Comment:
Instead of having a defined package name in the synopsis (and perpetuating not the best of code practices), it might be better to replace all the `Stuff` bits with `__PACKAGE__` instead.

Reply:
I agree! This is really just converting an existing README -> README.md but, if you make a case [here](https://rt.cpan.org/Public/Dist/Display.html?Name=Class-Data-Inheritable) I'll include it within the next release!